gdm_session_switch-system-indicator-classic-20160913.json

```diff
@@ -0,0 +1,18 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "type": "match",
+      "xpos": 986,
+      "ypos": 1,
+      "width": 36,
+      "height": 23
+    }
+  ],
+  "tags": [
+    "ENV-DESKTOP-gnome-classic",
+    "ENV-DISTRI-opensuse",
+    "ENV-INSTLANG-en_US",
+    "system-indicator"
+  ]
+}
```

Comment:
My local server has a low performance so nautilus is closed with a delay.
needle 'system-indicator' is only a small area in the top right corner of the desktop, the nautilus window is harmless.